### ISSUE TITLE
"runme" scripts have env-vars set differently than website

### ISSUE THREAD

**Issue by mauro3** [eth-vaw-glaciology/course-101-0250-00#16]
The website https://pde-on-gpu.vaw.ethz.ch/software_install/#cuda-aware_mpi_on_piz_daint states
```
export MPICH_RDMA_ENABLED_CUDA=1
export IGG_CUDAAWARE_MPI=1
```
but the linked scripts in https://github.com/eth-vaw-glaciology/course-101-0250-00/tree/main/scripts/l8_scripts set them `=0`.

Also, whilst at it, should the last line be `julia --project -O3 <my_script.jl>` ?

**Comment by luraess** [comment]
There is currently an issue with CUDA-aware MPI on Daint - see https://pde-on-gpu.vaw.ethz.ch/software_install/#cuda-aware_mpi_on_piz_daint

Scripts are ready to run l8 tasks. The `<my_script.jl>` is just a generic place holder.

**Comment by mauro3** [comment]
Ok, but then it's a bit confusing to have several places claiming to be identical doing different things.

**Comment by luraess** [comment]
> Ok, but then it's a bit confusing to have several places claiming to be identical doing different things.

Yes, Ideally we don't want this situation to last for too long, thus I did not change the infos everywhere. But yes, right now it could be confusing as different things are stated in different places.

**Comment by mauro3** [comment]
I also confused myself by having an old version of the webpage open without the red banner.

In general, would it be possible to have the code-snippets in the website picked up in the actual scripts.  That way only one place would need to be changed.

**Comment by luraess** [comment]
unsure if we can come up with a simple and efficient solution here... will think about it

**Comment by mauro3** [comment]
Let's discuss sometime.
